### PR TITLE
Add PID tracking to the ServiceFabric systemd services

### PR DESCRIPTION
**Pull request by daviddob**
Adds PID tracking to the `servicefabric` and `servicefabricupdater` processes that are run by systemd for use in monitoring and other scripts. It modifies the systemd services to drop pidfiles in `/var/run/` in accordance with other services.

### CODE REVIEW

**Comment by vturecek** [comment]
@daviddob thanks! 
@bpm-ms can you take a look at this?

File: src/prod/linuxsetup/runtime/scripts/servicefabricupdater.service [hunk 1]
```diff
@@ -5,6 +5,7 @@ Description=ServiceFabric Updater Daemon
 Type=simple
 EnvironmentFile=/etc/servicefabric/servicefabricupdater.config
 ExecStart=/opt/microsoft/servicefabric/bin/doupgrade.sh $PathToTargetInfoFile
+ExecStartPost=/bin/sh -c 'umask 022; pgrep starthost.sh > /var/run/servicefabricupdater.pid'
```
Comment (bpm-ms) [inline src/prod/linuxsetup/runtime/scripts/servicefabricupdater.service]: Should this be `doupgrade.sh` instead of `starthost.sh`?

Comment (daviddob) [reply src/prod/linuxsetup/runtime/scripts/servicefabricupdater.service]: It should, I apparently had it as a local change that didn't get committed.

Comment (anmolah) [reply src/prod/linuxsetup/runtime/scripts/servicefabricupdater.service]: +1

**Comment by bpm-ms** [comment]
Looks good! Thanks for the contribution!

**Pull request merged**